ecmascript/transforms/module/src/common_js.rs

```diff
@@ -372,6 +372,28 @@ where
                                     // init_export!("default");
                                     let ident = ident.unwrap_or_else(|| private_ident!("_default"));
 
+                                    let member_expr = member_expr!(DUMMY_SP, exports.default);
+                                    match &*member_expr {
```

Comment:
As value of `member_expr` is known, I think this can be improved for better readability.

Reply:
Good point, thanks. maybe this works? https://github.com/swc-project/swc/pull/2705/files#diff-506530d02689c3c49a06cbec6fb8c9ffd7b1df4c551dcac29d8bd08d0248590fR380

Reply:
Yes, thanks!